ee/maintained-apps/outputs/apps.json

```diff
@@ -0,0 +1,164 @@
+{
+  "apps": [
+    {
+      "version": 2,
```

Comment:
Should be at the top level of the manifest, not per-app.

Reply:
misread the example!

Reply:
misread the example!